### DISCUSSION TITLE
Deleting the discussion category problem

### DISCUSSION THREAD

**Discussion by shiva-sandupatla** [community/community#3171] [Discussions] [answered]
In a public repo, when I try to delete a category, it is not responding and redirecting to `https://github.com/BlockySite-Community/BlockySite-News/discussions/categories/32361848` . 

Hoping with a positive response

Regards, @Shiva-sandupatla

**Comment by evi-liu** [comment, marked as answer]
Hi there! This was a bug we uncovered, but it should be fixed now :). Let me know if you're still seeing this behavior.

**Reply by shiva-sandupatla** [reply]
Thanks a lot for fixing it!!
